starter-code/src/components/ProductRow.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+
+class ProductRow extends Component {
+  render() {
+    return (
+      <tr>
+        <td style={{ color: this.props.stocked ? "" : "red" }}>
```

Comment:
Cette expression ternaire peut être remplacée par `this.props.stocked || "red"` ou bien `!this.props.stocked && "red"`